lib/utils/index.ts

```diff
@@ -750,6 +751,29 @@ export function quantToString(quant: Readonly<Quant>): string {
     return value
 }
 
+/**
+ * Returns a regexp literal text of the given char set.
+ */
+export function charSetToRegExpText(
```

Comment:
I think this function should be renamed. I mean, what is "regexp text"? "Text" might refer to a whole regexp literal (e.g. `/foo/i`) or just its source (e.g. `foo`). We should replace "text" with "source".

However, we can be even more specific than "source". `JS.toLiteral` guarantees that it produces a literal representation of the given RE AST. Since we only give it a single character class, it is guaranteed to produce a single character, character class, or character set. (I should have mentioned that in the doc.)

```suggestion
export function toCharSetSource(
```